include/config.h

```diff
@@ -36,4 +36,14 @@
 #endif
 #endif
 
+// Compatibility definition for other projects to detect pokeemerald-expansion
+#define RHH_EXPANSION
+
+// Legacy branch-based defines included for backwards compatibility
+#ifdef RHH_EXPANSION
```

Comment:
Don't know if this ifdef is even needed.

Reply:
Was thinking about that in retrospect, it does seem redundant. I think I was initially considering if someone wanted to "turn off" all of the defines at once without commenting them out, but that's pretty trivial to do, and I'm not sure why anyone would want to do that in the first place.

Feel free to remove the ifdef, or I can update with a commit if you want.

Reply:
Please do.

Reply:
Fixed 👍